attributes/plugins.rb

```diff
@@ -6,21 +6,21 @@
   "ant" => "1.9",
   "antisamy-markup-formatter" => "1.5",
   "apache-httpcomponents-client-4-api" => "4.5.10-2.0",
-  "audit-trail" => "3.5",
+  "audit-trail" => "3.7",
   "badge" => "1.6",
   "bazaar" => "1.22",
   #"benchmark" => "1.0.12-SNAPSHOT (private-50491052-cottsay)",
   "bootstrap4-api" => "4.4.1-11",
   "bouncycastle-api" => "2.17",
   "branch-api" => "2.0.20",
   "build-timeout" => "1.19",
-  "cloudbees-folder" => "6.6",
+  "cloudbees-folder" => "6.14",
   "collapsing-console-sections" => "1.8.0",
-  "configuration-as-code" => "1.43",
   "command-launcher" => "1.2",
   "conditional-buildstep" => "1.3.6",
   "config-file-provider" => "3.6.2",
-  "copyartifact" => "1.41",
+  "configuration-as-code" => "1.43",
+  "copyartifact" => "1.45.2",
```

Comment:
There is a disruptive change in 1.44 of copy artifact, [see changelog](https://github.com/jenkinsci/copyartifact-plugin/blob/master/CHANGELOG.adoc) although a migration mode is enabled by default and things should keep working. Not sure if you need to do any action.

Reply:
Yeah I've already looked at the migration mode. It would have been smart of me to mention as much in the PR description :+1:. I'll add info there. We're good in migration mode for now.

Reply:
Updated the PR description.